mlflow/store/artifact/ftp_artifact_repo.py

```diff
@@ -112,6 +112,8 @@ def list_artifacts(self, path=None):
                 return []
             artifact_files = ftp.nlst(list_dir)
             artifact_files = list(filter(lambda x: x != "." and x != "..", artifact_files))
+            # Make sure artifact_files is a list of file names because ftp.nlst may return absolute paths.
```

Comment:
@shaneing Can you wrap this line to fix lint error?

Reply:
Hang on a moment. I have installed `pylint`.